package.json

```diff
@@ -30,6 +30,7 @@
     "angular-cli-ghpages": "^1.0.0-rc.1",
     "antlr4": "^4.7.2",
     "core-js": "^2.6.2",
+    "date-format": "^3.0.0",
```

Comment:
Why do you use a library for the time format even though you format everything else yourself?

Reply:
good point, removed.